source/_components/light.limitlessled.markdown

```diff
@@ -80,3 +80,129 @@ Refer to the [light]({{site_root}}/components/light/) documentation for general
+  - Lounge & Hall, both are bound to zone 2
+  - Study, bound to zone 3
+  - Bedroom, bound to zone 4
+
+Other Tips I found on an article on the LimitlessLED website that might be of use:
+
+We also had some feedback/tips from users:
+1. Tip#10. I had to use the LimitlessLED v1.7 app to connect the bridge to my LAN, then it worked, thanks.
+1. Tip#11. I had to use the correct remote screen in the app for my particular RGBW LEDs to sync, thanks.
+1. Tip#12. I had to clear remote code pairing using section 3.3.5 of the manual, and then the app and wifi
+bridge worked. Thx
+1. Tip#13. “I figured it out. I was using a fixture that held multiple bulbs. Apparently there is interference
+when you try to pair two bulbs at the same time. A fixture with a single bulb worked.”
+1. Tip#14. It worked after I changed my wifi router to the standard wifi secure settings WPA2/PSK AES.
+1. Tip#15. I was able to connect the bridge to my network after I set the SSID to visible.
+1. Tip#16. I placed the wifi bridge closer to the wifi router.
+1. Tip#17. After trying another usb power supply to the wifi bridge it worked.
+1. Tip#18. Might be some sneaky power leaking on to your socket with the switch even off. So remove
+the bulb from the socket, turn the switch on at the wall. Plug the bulb to the socket, and within 2
+seconds press the group1 on button on the remote. Dual white remote for dual white bulbs, RGBW
+remote for rgbw bulbs. Now for the App, you must sync it again separately, because the wifi bridge acts
+just like its own remote.
+
+
+### Add the config to the configuration.yaml file
+
+Here's my working light section:
+
+    light:
+      platform: limitlessled
+      bridges:
+        - host: 192.168.0.138
+          version: 6
+          port: 5987
+          groups:
+          - number: 1
+            type: white
+            name: Kitchen
+          - number: 1
+            type: bridge-led
+            name: Bridge Light
+          - number: 1
+            type: rgbw
+            name: Kitchen table
+          - number: 2
+            type: rgbw
+            name: Lounge & Hall
+          - number: 3
+            type: rgbw
+            name: Study
+          - number: 4
+            type: rgbw
+            name: Bedroom
+
+Then reboot your Pi.
+You should now see the lights on your home page in HomeAssistant.
+
+Things to note here...
+
+1. not sure if the version and port are required (docs say it's optional), but I added for completeness and to remind me if I ever upgrade or run into issues
+1. There are a total of four groups because that's all that the v6 bridge can handle... however... on my main page of HomeAssistant I have **6 independently controllable entities**, i.e. there are 6 on/off switches listed. This is because each of the 4 number/groups here can have duplicates (see I have three 'number 1' entities) because each of those are a different 'type'; a 'white', a 'rgbw', and the 'bridge-led'.
```

Comment:
**Spelling & Grammar**:
- Remove/replace the `...`.
- `HomeAssistant` -> `Home Assistant`
- Concern:  `I have`: There is no reference to the author, rewrite please.
- Missing comma after `i.e.` -> `i.e., there are `
- Form of verb `those are a different` -> `those is a different`
- `rgwb` -> `RGWB`